package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@api-components/api-server-selector",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
It's OK to do this but before the final release we will have to upgrade this to at least `0.1.0` and then each component that uses this component to use new minor version. Just be aware of that.